src/drivers/MemoryDriver.ts

```diff
@@ -0,0 +1,57 @@
+import { IDriver } from "./IDriver";
+
+type Table = Map<string, any>;
+
+export class MemoryDriver implements IDriver {
+    public $store = new Map<string, Table>();
+
+    public async prepare(table: string): Promise<void> {
+        this.$getOrCreateTable(table);
+    }
+
+    public $getOrCreateTable(name: string) {
```

Comment:
Same here with the dollar sign prefix.
You can remove the prefix

Even thought the return type of the function is inferred by typescript, I would suggest adding the return type so nothing else that doesn't fit this type can be returned